grow/commands/build.py

```diff
@@ -18,8 +18,11 @@
 @click.option('--clear_cache',
               default=False, is_flag=True,
               help='Clear the pod cache before building.')
-@click.option('--file', '--pod_path', 'pod_paths', help='Build only pages affected by content files.', multiple=True)
-def build(pod_path, out_dir, preprocess, clear_cache, pod_paths):
+@click.option('--file', '--pod-path', 'pod_paths', help='Build only pages affected by content files.', multiple=True)
+@click.option('--display-untranslated',
```

Comment:
I've renamed it to `--locate-untranslated`. Then it is a bit more broad than just display and we can use it to create the `.po` file as needed.